ContributionGuidelines.md

```diff
@@ -0,0 +1,140 @@
+# [DOC2PEN](https://doc2pen.smaranjitghose.codes/)📝
+#### Doc2pen is an Open Source Software that converts your Typed Docs into Handwritten Docs.
+
+# Contribution Guidelines [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/dwyl/esta/issues) 🤝🏽🍀:
```

Comment:
Make the title please

Reply:
I don't understand

Reply:
Keep the first line of the file as #Contribution Guidelines